crates/ruff_python_formatter/resources/test/fixtures/black/py_312/type_params.py

```diff
@@ -0,0 +1,13 @@
+def func  [T ](): pass
+async def func [ T ] (): pass
+class C[ T ] : pass
+
+def all_in[T   :   int,U : (bytes, str),*   Ts,**P](): pass
```

Comment:
We probably want a "long" case with type var tuples (`*Ts`) and (`**P`) to cover splitting behavior with the leading stars.